mbzirc_ign/src/Surface.cc

```diff
@@ -63,6 +66,12 @@ class ignition::gazebo::systems::SurfacePrivate
 
   /// \brief The world's gravity [m/s^2].
   public: ignition::math::Vector3d gravity;
+
+  /// \brief Wavefield pointer.
+  public: std::unique_ptr<Wavefield> wavefield;
```

Comment:
does this need to be a unique pointer or can it just be `public: Wavefiled wavefield;`?

Reply:
Not at all. Updated in [47efdd9](https://github.com/osrf/mbzirc/pull/31/commits/47efdd99a68f9c014185c531313ea7d4c505d760)